src/core/search/searchdatatransformer.js

```diff
@@ -46,7 +46,7 @@ export default class SearchDataTransformer {
       [StorageKeys.QUERY_ID]: response.queryId,
       [StorageKeys.NAVIGATION]: new Navigation(), // Vertical doesn't respond with ordering, so use empty nav.
       [StorageKeys.VERTICAL_RESULTS]: VerticalResults.fromCore(
-        response.verticalResults, {}, resultsContext),
+        response.verticalResults, {}, resultsContext, verticalKey),
       [StorageKeys.DYNAMIC_FILTERS]: DynamicFilters.from(response.facets, resultsContext),
```

Comment:
Do we need to do any kind of transformation for the SDK to properly parse the facets returned by the Core library?

Reply:
Yes, we do! This PR is only really for the facets request, the response parsing changes are in this PR: https://github.com/yext/answers/pull/1193